.github/workflows/webviz-core-components.yml

```diff
@@ -78,3 +78,16 @@ jobs:
           python -m pip install --upgrade setuptools wheel twine
           python setup.py sdist bdist_wheel
           twine upload dist/*
+
+      - name: 🗄️ Setup .npmrc file to publish to npm
+        if: github.event_name == 'release' && matrix.python-version == '3.6'
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
What are pros/cons of creating `.npmrc` during CI vs https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#creating-a-scoped-public-package and commiting `.npmrc` to the repo?